src/main/java/mate/academy/internet/shop/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.internet.shop.dao.impl;
+
+import java.util.List;
+import java.util.Optional;
+
+import mate.academy.internet.shop.database.Storage;
+import mate.academy.internet.shop.lib.Dao;
+import mate.academy.internet.shop.model.Order;
+
+@Dao
+public class OrderDaoImpl implements mate.academy.internet.shop.dao.OrderDao {
```

Comment:
```suggestion
public class OrderDaoImpl implements OrderDao {
```
треба, щоб так виходило :)

Reply:
Sorry)